sdk/bootstrap/build.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# exit immediately on failure
+set -e
```

Comment:
`set -eu` to also disallow the reference of unset envvars? I'd normally also plug `-o pipefail` but there are no pipes in this script.